test/unit/mongo/MongoTripodConfigTest.php

```diff
@@ -1707,6 +1707,17 @@ public function testGetResqueServer()
         $this->assertEquals(getenv(MONGO_TRIPOD_RESQUE_SERVER), \Tripod\Mongo\Config::getResqueServer());
     }
 
+    public function testGetResqueDatabase()
+    {
+        \Tripod\Mongo\Config::setValidationLevel(\Tripod\Mongo\Config::VALIDATE_MAX);
```

Comment:
I don't think this is needed? Copied from the test above but that also looks outdated.